barretenberg/sol/src/honk/BaseHonkVerifier.sol

```diff
@@ -45,7 +45,8 @@ abstract contract BaseHonkVerifier is IVerifier {
     error ShpleminiFailed();
 
     // Number of field elements in a ultra honk  proof
-    uint256 constant PROOF_SIZE = 440;
+    uint256 constant PROOF_SIZE = 456;
```

Comment:
can you please add a comment explaining whether the pairing point is part of the proof